packages/main/src/WheelSlider.js

```diff
@@ -218,49 +208,50 @@ class WheelSlider extends UI5Element {
 		};
 	}
 
-	_handleWheel(e) {
-		if (!e) {
-			return;
-		}
+	expandSlider() {
+		this._expanded = true;
+		this.fireEvent("expand", {});
+	}
 
-		e.stopPropagation();
-		e.preventDefault();
+	collapseSlider() {
+		this._expanded = false;
+		this.fireEvent("collapse", {});
+	}
 
-		if (e.timeStamp === this._prevWheelTimestamp || !this._expanded) {
-			return;
-		}
+	_updateItemCellHeight() {
+		this._itemCellHeight = this.shadowRoot.querySelectorAll(".ui5-wheelslider-item").length && Number(getComputedStyle(this.shadowRoot.querySelector(".ui5-wheelslider-item")).getPropertyValue("--_ui5_wheelslider_item_height").replace("rem", ""));
```

Comment:
This line is really hard to assimilate, one has to break it down in their mind first. Please split to several lines and document it by assigning the sub-expressions to variables.